tests/lib/RichText/Converter/Xslt/_fixtures/xhtml5/edit/037-orderedListFormatted.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<section xmlns="http://ibexa.co/namespaces/ezpublish5/xhtml5/edit">
+    <p>default</p>
+    <ol>
+        <li>default 1</li>
+        <li>default 2</li>
+    </ol>
+    <p>decimal</p>
+    <ol style="list-style-type:decimal;">
+        <li>decimal 1</li>
+        <li>decimal 2</li>
+    </ol>
+    <p>lower roman</p>
+    <ol style="list-style-type:lower-roman;">
+        <li>lower roman 1</li>
+        <li>lower roman 2</li>
+    </ol>
+    <p>upper roman</p>
+    <ol style="list-style-type:upper-roman;">
+        <li>upper roman 1</li>
+    </ol>
+    <ol>
+        <li>upper roman 2</li>
+    </ol>
+    <p>lower latin</p>
+    <ol style="list-style-type:lower-latin;">
+        <li>lower latin 1</li>
+        <li>lower latin 2</li>
+    </ol>
+    <p>upper latin</p>
+    <ol style="list-style-type:upper-latin;">
+        <li>upper latin 1</li>
+        <li>upper latin 2</li>
+    </ol>
+</section>
```

Comment:
missing CRLR at the end of file.

Reply:
Fixed in f7d30bfbc903411207f2739be293b5af84fd4a8a